src/components/elements/Tabs/Default/TabButton.tsx

```diff
@@ -34,21 +34,22 @@ export const TabButton = ({
       className={classNames(
         className,
         `w-full items-center focus:outline-none disabled:text-neutral-900`,
-        "border",
+        "",
```

Comment:
I think this empty string isn't accomplishing anything, please remove.

Reply:
done